gradle.properties

```diff
@@ -20,4 +20,6 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
It is better to move declaring buildconfig in build.gradle (:app level) module. [Reasons and consequences](https://stackoverflow.com/a/74692390/20937549).

Reply:
You might also be interested in using [buildConfigField](https://developer.android.com/build/gradle-tips#share-custom-fields-and-resource-values-with-your-app-code) later on in your tasks.